### PR TITLE
Fix dev env setup issues 

### PR DESCRIPTION
**Pull request by peter-ni-noob**
## Proposed change

Addressing environmental set-up issues of this projects for new contributors, command "make package" is useful to set up environment of this project when begining this project or updating the version of project for avertting error when "make test".

## Type of change

- [ ] New country/market holidays support (thank you!)
- [ ] Supported country/market holidays update (calendar discrepancy fix, localization)
- [x] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've run `make pre-commit`, it didn't generate any changes
- [x] I've run `make test`, all tests passed locally

[contributing-guidelines]: https://github.com/vacanza/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/python-holidays/tree/beta/docs/source

### CODE REVIEW

**Comment by peter-ni-noob** [comment]
Also ,I solved the issue in #1603

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ holidays/locale/pot/*.pot
 MANIFEST
 Pipfile
 venv/
+mytest.py
```
Comment (KJhellico) [inline .gitignore:23]: Looks like it's some private file.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -55,13 +55,13 @@ repos:
     hooks:
       - id: rst-backticks
 
-  - repo: https://github.com/myint/rstcheck
```
Comment (KJhellico) [inline .pre-commit-config.yaml:58]: We can't just remove this test.

File: scripts/generate_release_notes.py [hunk 3]
```diff
@@ -77,11 +77,19 @@ def __init__(self) -> None:
             type=str,
         )
         arg_parser.add_argument(
```
Comment (KJhellico) [inline scripts/generate_release_notes.py:79]:
```python
        arg_parser.add_argument(
            "-v",
            "--verbose",
            action="store_true",
            default=False,
            help="Verbose output",
        )

```

File: .gitignore (hunk 1 above)
Comment (peter-ni-noob) [reply .gitignore]: I have restored to origin

File: .pre-commit-config.yaml (hunk 2 above)
Comment (peter-ni-noob) [reply .pre-commit-config.yaml]: I have restored to origin

File: scripts/generate_release_notes.py (hunk 3 above)
Comment (peter-ni-noob) [reply scripts/generate_release_notes.py:79]: I have taken your advice.

File: scripts/generate_release_notes.py [hunk 4]
```diff
@@ -79,9 +79,11 @@ def __init__(self) -> None:
...
             "-v",
             "--verbose",
-            action=argparse.BooleanOptionalAction,
+            action="store_true",
+            default=False,
             help="Verbose output",
         )
+
```
Comment (KJhellico) [inline scripts/generate_release_notes.py:86]:
```suggestion
```

**Review by KJhellico — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -36,6 +36,7 @@ First step is setting up development environment and installing all the required
     $ source venv/bin/activate
 
     $ make setup
+    $ make package
```
Comment (arkid15r) [inline CONTRIBUTING.rst:39]: This should belong in the Makefile, just add this command [here](https://github.com/vacanza/python-holidays/blob/58cd972d6a5f96ab170241945504ecf878a96925/Makefile#L52)

**Review by arkid15r — COMMENTED** [review]
👍 Thanks for addressing the issues

File: CONTRIBUTING.rst (hunk 5 above)
Comment (peter-ni-noob) [reply CONTRIBUTING.rst]: OK, I have followed your advice.

**Review by arkid15r — APPROVED** [review]
Merging this, thank you @peter-ni-noob !

upd: it seems this requires an explicit @KJhellico's approval: `Merging can be performed automatically once the requested changes are addressed.`

**Review by KJhellico — APPROVED** [review]
LGTM.

**Pull request merged**